providers/src/airflow/providers/openlineage/extractors/manager.py

```diff
@@ -36,8 +36,8 @@
 if TYPE_CHECKING:
     from openlineage.client.event_v2 import Dataset
 
-    from airflow.lineage.entities import Table
     from airflow.models import Operator
+    from airflow.providers.common.compat.lineage.entities import Table
```

Comment:
Please bump the minimum version of providers common.compat requirement in `provider.yaml` in this case. This would raise exception for previous version of common.compat dependency.